xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -198,16 +231,9 @@ viewers.Comments = Class.create({
    * Inline reply: Move the form under the replied comment and update the hidden "replyto" field.
    */
   addReplyListener : function() {
-    if (this.form) {
-      $$(this.xcommentSelector).each(function(item) {
-        this.addReplyListenerToComment(item);
-      }.bind(this));
-    } else {
-      // If, for some reason, the form is missing, hide the reply functionality from the user
-      $$(this.xcommentSelector + ' a.commentreply').each(function(item) {
-        item.hide();
-      });
-    }
```

Comment:
I guess you removed this if because it shouldn't be needed anymore with your getForm, is that correct?